hotkey.go

```diff
@@ -119,6 +119,61 @@ func (hk *Hotkey) String() string {
 	return s
 }
 
+var namedKeyMap map[string]Key
+
+//run at run-time
+func init() {
+	namedKeyMap = map[string]Key{
+		"0":      Key0,
+		"1":      Key1,
+		"2":      Key2,
+		"3":      Key3,
+		"4":      Key4,
+		"5":      Key5,
+		"6":      Key6,
+		"7":      Key7,
+		"8":      Key8,
+		"9":      Key9,
+		"Space":  KeySpace,
+		"a":      KeyA,
+		"b":      KeyB,
+		"c":      KeyC,
+		"d":      KeyD,
+		"e":      KeyE,
+		"f":      KeyF,
+		"g":      KeyG,
+		"h":      KeyH,
+		"i":      KeyI,
+		"j":      KeyJ,
+		"k":      KeyK,
+		"l":      KeyL,
+		"m":      KeyM,
+		"n":      KeyN,
+		"o":      KeyO,
+		"p":      KeyP,
+		"q":      KeyQ,
+		"r":      KeyR,
+		"s":      KeyS,
+		"t":      KeyT,
+		"u":      KeyU,
+		"v":      KeyV,
+		"w":      KeyW,
+		"x":      KeyX,
+		"y":      KeyY,
+		"z":      KeyZ,
+		"Return": KeyReturn,
+		"Escape": KeyEscape,
+		"Delete": KeyDelete,
+		"Tab":    KeyTab,
+	}
+}
+
+//Parses a Key from a string, returns if the key was found as the second value
+func ParseKey(name string) (Key, bool) {
+	value, ok := namedKeyMap[name]
+	return value, ok
+}
```

Comment:
I don't know if this should be offered in a package. I have the tendency to leave this work for the API user. If we defined string to key code conversion, people will start to ask more and more, to extend this table, which creates endless releases.

Reply:
I agree that this library could do without the `ParseKey` function, because then people will start to ask for more features.
But I think that it is also quite tedious to name all keys by e.g.: defining `namedKeyMap` as a user. I would like some kind of way to name keys, so that every key has an assigned string. Then users would be able to implement their own parsing of strings to Keys, Hotkeys, etc..
I don't know if there is some other kind of way to implement this in Go.
The map would only be extended if new keys would be added.

Reply:
How about I remove `ParseKey`?

Also some kind of `namedModMap` would also make sense.

Reply:
Yeah I think it would be better if we just remove this.

Reply:
Removed `ParseKey`. I added the modifier map similar to the `namedKeyMap`. The variables now start with uppercase letters, so that they can be accessed from outside the package.